resources/startNativeNotebooks.ipynb

```diff
@@ -6,7 +6,25 @@
    "source": [
     "# Welcome to the New VS Code Notebook Experience! \r\n",
     "\r\n",
-    "## To learn more about the new Notebook experience and get a tour around, click here: https://github.com/microsoft/vscode-jupyter/wiki/Introducing-Native-Notebooks"
+    "Whether you're new to the notebook experience in VS Code or a well-seasoned user, we'd like you to know that our experience has been revamped to provide:\r\n",
```

Comment:
```suggestion
    "Whether you're new to the notebook experience in VS Code or a seasoned user, we'd like you to know that our experience has been revamped to provide:\r\n",
```

🧂?

Reply:
Yeah, I don't even like seasoned... But I'm blanking on another term. But for now, I've removed 'well-' :)